distro/pom.xml

```diff
@@ -16,19 +16,50 @@
   <packaging>pom</packaging>
 
   <properties>
-    <openmrs.version>2.7.4</openmrs.version>
-    <webservices.rest.version>2.49.0</webservices.rest.version>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <maven.compiler.source>8</maven.compiler.source>
     <maven.compiler.target>8</maven.compiler.target>
     <limeEmrVersion>0.1.0</limeEmrVersion>
-    <referenceApplicationDockerVersion>3.4.0-rc.1</referenceApplicationDockerVersion>
```

Comment:
Does this mean we've decided to switch from using the docker versions to loading the modules individually?